Libplanet.Tests/Blocks/BlockTest.cs

```diff
@@ -275,15 +275,16 @@ public void EvaluateActions()
                     }
                 ),
             };
+
             Block<DumbAction> blockIdx2 = MineNext(blockIdx1, blockIdx2Txs);
             pairs = blockIdx2
                 .EvaluateActionsPerTx(dirty1.GetValueOrDefault)
                 .ToImmutableArray();
             expectations = new[]
             {
-                (0, 0, new[] { "A,D", "B", "C", null, null }),
-                (1, 0, new[] { "A,D", "B", "C", "E", null }),
-                (2, 0, new[] { "A,D", "B", "C", "E", "RecordRehearsal:False" }),
+                (0, 0, new[] { "A,D", "B", "C", null, null }, _fx.TxFixture.PrivateKey1.PublicKey.ToAddress()),
+                (1, 0, new[] { "A,D", "B", "C", "E", null }, _fx.TxFixture.PrivateKey2.PublicKey.ToAddress()),
+                (2, 0, new[] { "A,D", "B", "C", "E", "RecordRehearsal:False" }, _fx.TxFixture.PrivateKey3.PublicKey.ToAddress()),
```

Comment:
As it's likely to use this pattern more and more, how about having `TxFixture.Address2` and `TxFixture.Address3`?

Reply:
Added. (cf5eedbe757b312358635dec47e565ae0b46e671)